src/SmartComponents/modules/reducers.js

```diff
@@ -144,6 +214,7 @@ function compareReducer(state = initialState, action) {
                 stateFilter: action.payload,
                 page: 1,
                 filteredCompareData: paginatedFacts,
+                dataToConvert: sortedFacts,
```

Comment:
Should we store this as `sortedFilteredFacts` ? I find `dataToConvert` not as obvious as to what it means.